books/free-programming-books.md

```diff
@@ -1961,8 +1961,6 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Learning React.js: Getting Started and Concepts](https://scotch.io/tutorials/learning-react-getting-started-and-concepts)
 * [React-Bits](https://github.com/vasanthk/react-bits)
 * [React Book, your beginner guide to React](https://github.com/softchris/react-book/) - Chris Noring
-* [React Enlightenment](https://www.gitbook.com/book/frontendmasters/react-enlightenment/details)
-* [React In-depth: An exploration of UI development](https://www.gitbook.com/book/developmentarc/react-indepth/details)
```

Comment:
I think the link for this one has just changed is all:

* https://developmentarc.gitbooks.io/react-indepth/content/

Reply:
I added it in the next commit